lib/pages/chat_list/chat_list_item.dart

```diff
@@ -201,140 +205,189 @@ class ChatListItem extends StatelessWidget {
                           ),
                       ],
                     ),
-                  ),
-                  Padding(
-                    padding: const EdgeInsets.only(left: 4.0),
-                    child: Text(
-                      room.timeCreated.localizedTimeShort(context),
-                      style: Theme.of(context).textTheme.labelSmall,
-                    ),
-                  )
-                ],
-              ),
-              subtitle: Row(
-                mainAxisAlignment: MainAxisAlignment.center,
-                children: <Widget>[
-                  if (typingText.isEmpty &&
-                      ownMessage &&
-                      room.lastEvent!.status.isSending) ...[
-                    const SizedBox(
-                      width: 16,
-                      height: 16,
-                      child: CircularProgressIndicator.adaptive(strokeWidth: 2),
-                    ),
-                    const SizedBox(width: 4),
+                    isGroup
```

Comment:
IMO, i think if code is nested too much, you should separate it to a helper function, or a widget